### DISCUSSION TITLE
nodered container crashes after update

### DISCUSSION THREAD

**Discussion by heinzgert** [ct-Open-Source/ct-Smart-Home#69] [Q&A]
Today I updated my ct-Smart-Home installation on a Raspi 4 today via the start.sh update function. Since then, the nodered container does not start anymore and I get this output in the logfile:

nodered_1  | Starting dbus daemon
nodered_1  | Starting Avahi daemon
nodered_1  | Starting Node-Red
nodered_1  | 
nodered_1  | 
nodered_1  | #
nodered_1  | # Fatal error in , line 0
nodered_1  | # unreachable code
nodered_1  | #
nodered_1  | #
nodered_1  | #
nodered_1  | #FailureMessage Object: 0xbef6124c
nodered_1  | Illegal instruction

Before the udpate, everything worked smoothly ...
Has anybody experienced this before and can help here?
After hours of searching and trying, I'm kind of desperated, any idea or help is welcome!

**Comment by heinzgert** [comment]
When switching back the nodered image to "latest-10", I can start again, so my system is up again :-)
However, "latest" and and "latest-12" results in the crash. I also tried to start from scratch with a complete new clone of the project, same result.

**Comment by T1ct4c** [comment]
Hi,

i´ve got the same issue after updating via ./start.sh update.
Zigbee Interface is running and available.
